src/dbg/debugger_commands.cpp

```diff
@@ -142,6 +142,11 @@ CMDRESULT cbDebugRun(int argc, char* argv[])
     // Don't "run" twice if the program is already running
     if(dbgisrunning())
         return STATUS_ERROR;
+    if(!ThreaderIsAnyThreadActive())

```

Comment:
I think this code has to be removed, the 'running' state is true its just that `WaitForDebugEvent` will be stuck until a thread resumes. Maybe we can just warn the user, I think the suspend count is available from the thread module in x64dbg...